app/loggers/log_worker_queue_stats.rb

```diff
@@ -0,0 +1,30 @@
+class LogWorkerQueueStats
+  def self.perform
+    record_totals
+    record_queue_stats
+  end
+
+  private
+
+  def self.record_totals
+    log_to_datadog("sidekiq.queues.total_size", Sidekiq::Queue.all.map(&:size).sum)
```

Comment:
Don't know how expensive this operation is but maybe we can call `Sidekiq::Queue.all` only once and pass the result to the two private methods as argument, since I see it's called again on line 16

Reply:
Doesn't really matter how expensive it is, one less call is always better!!! 🎉